src/main/resources/application.properties

```diff
@@ -25,6 +25,9 @@ healthInformationConsentManagerPath=/v0.5/health-information/cm/request
 healthInformationOnRequestPath=/v0.5/health-information/hip/on-request
 healthInformationPushNotificationPath=/v0.5/health-information/notify
 
+# Share profile
+profileOnSharePath=/v1.0/patients/profile/on-share
```

Comment:
where is this being used?

Reply:
The share profile related code will be added in the upcoming pr's.